examples/js/postprocessing/RenderPass.js

```diff
@@ -39,7 +39,7 @@ THREE.RenderPass.prototype = Object.assign( Object.create( THREE.Pass.prototype
 
 		}
 
-		renderer.render( this.scene, this.camera, readBuffer, this.clear );
+		renderer.render( this.scene, this.camera, this.renderToScreen ? null : readBuffer, this.clear );
```

Comment:
Whether one writes to read buffer or write buffer depends on whether one
sets swapbuffers to be true or not.  I guess this code always assumes that
one is not swaping buffers.

Best regards,
Ben Houston (Cell: 613-762-4113, Skype: ben.exocortex, Twitter:
@exocortexcom)
https://Clara.io - Online 3D Modeling and Rendering

On Sun, Jun 12, 2016 at 4:36 PM, Mr.doob <notifications@github.com> wrote:

> In examples/js/postprocessing/RenderPass.js
> <https://github.com/mrdoob/three.js/pull/9094#discussion_r66726786>:
>
> > @@ -39,7 +39,7 @@ THREE.RenderPass.prototype = Object.assign( Object.create( THREE.Pass.prototype
> >
> >  		}
> >
> > -		renderer.render( this.scene, this.camera, readBuffer, this.clear );
> > +		renderer.render( this.scene, this.camera, this.renderToScreen ? null : readBuffer, this.clear );
>
> Oh wait. I guess this pass doesn't need a readBuffer and it renders
> directly to it.
> It's confusing though...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrdoob/three.js/pull/9094/files/8e08c07a198f0865b89cedac6fdb44791a829e1b#r66726786>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAj6_fIswIPg2gFnmkpMaiRllvVNnxEDks5qLG3fgaJpZM4Iv97b>
> .
>